gtk/transmission-gtk.metainfo.xml.in

```diff
@@ -29,6 +29,14 @@ Copyright 2017 Endless Mobile, Inc.
     </p>
   </description>
   <url type="homepage">https://transmissionbt.com/</url>
+  <url type="bugtracker">https://github.com/transmission/transmission/issues</url>
+  <url type="faq">https://transmissionbt.com/help/gtk/4.0x/html/FAQ.html</url>
+  <url type="help">https://transmissionbt.com/help/gtk/4.0x/index.html</url>
```

Comment:
```suggestion
  <url type="faq">https://transmissionbt.com/help/gtk/latest/html/FAQ.html</url>
  <url type="help">https://transmissionbt.com/help/gtk/latest/index.html</url>
```

https://github.com/transmission/transmission.github.io/pull/84 adds a symbolic link so that we can update it on the website w/o having to do anything with the xml here.